### ISSUE TITLE
Windows 11 enhanced support tickets

### ISSUE THREAD

**Issue by hfiref0x** [hfiref0x/WinObjEx64#28]
**~20210601~** Fixed in 1.9.1
**MmUnloadedDrivers** cannot be found using existing pattern because it is no longer unique.

21996 **MiRememberUnloadedDriver**
```
PAGE:00000001407FA273 BA D0 07 00 00                                                  mov     edx, 7D0h
PAGE:00000001407FA278 B9 40 00 00 00                                                  mov     ecx, 40h
PAGE:00000001407FA27D 41 B8 4D 6D 44 54                                               mov     r8d, 54446D4Dh
PAGE:00000001407FA283 E8 68 A0 AF FF                                                  call    MiAllocatePool
PAGE:00000001407FA288 48 89 05 09 F9 42 00                                            mov     cs:MmUnloadedDrivers, rax
```

Prior duplicate code **MiCreatePebOrTeb**

```
PAGE:00000001406B15A0 BA D0 07 00 00                                                  mov     edx, 7D0h
PAGE:00000001406B15A5 EB DF                                                           jmp     short loc_1406B1586
```

**Comment by hfiref0x** [comment]
Several Windows 11 internal structures have changed a lot, 20210902

E.g.
DEVICE_MAP, OBJECT_SYMBOLIC_LINK.

Since WinObjEx64 displays them as structure dump it should handle this correctly.

**Comment by hfiref0x** [comment]
Closed by dcfed82af7d63cdc0e6636e25182554410bb95e4
